slackbridge/bots.py

```diff
@@ -108,4 +111,17 @@ def signedOn(self):
             self.join('#{}'.format(channel['name']))
 
     def post_to_irc(self, channel, message):
-        self.msg(channel, message)
+        self.msg(channel, self.__format_message(message))
+
+    def __format_message(self, message):
+        match_ids = re.findall(r'(<\@([A-Z0-9]{9,})\>)', message)
+        # If for some weird reason someone mentions the same person
+        # twice in one message, we avoid extra api calls
+        for replace, uid in set(match_ids):
+            user_info = next(
+                (user for user in self.users if user['id'] == uid), None)
```

Comment:
I feel like splitting this across lines this way is confusing, maybe do this instead?

```python
user_info = next(
    (user for user in self.users if user['id'] == uid),
    None,
)
```